server/packaging/fedora/oauth-ssh.spec.in

```diff
@@ -30,6 +30,7 @@ BuildRequires:  libtool-ltdl-devel
 BuildRequires:  checkpolicy
 BuildRequires:  policycoreutils-python
 BuildRequires:  python-setuptools
+BuildRequires:  scitokens-cpp
```

Comment:
The scitokens-cpp package can be built from source (https://github.com/scitokens/scitokens-cpp) or RPMs are available from OSG (https://opensciencegrid.org/docs/common/yum/) and soon from EPEL (https://src.fedoraproject.org/rpms/scitokens-cpp).